market/market/coingecko/coingecko.go

```diff
@@ -69,6 +70,11 @@ func (m *Market) normalizeTicker(price coingecko.CoinPrice, provider string) (ti
 		} else if len(cg.TokenId) > 0 {
 			tokenId = cg.TokenId
 		}
+
+		if !isSuitableTicker(price.MarketCap, price.TotalVolume, price.CirculatingSupply) {
+			continue
+		}
```

Comment:
nit: these `continue` statements are kinda hard to control/read. How do you feel about leaving as is and introducing a new `filterTickers` function?

In your test you are also calling this `withFilters` so that would be in line with your test.

Reply:
Good idea, will be resolved soon